docs/REST API/rest-api.md

```diff
@@ -7711,11 +7821,176 @@ Deletes the session specified by ID. When the user associated with the session n
+
+#### Example
+
+`POST /api/v1/fleet/software/package`
+
+##### Request header
+
+```http
+Content-Length: 8500
+Content-Type: multipart/form-data; boundary=------------------------d8c247122f594ba0
+```
+
+##### Request body
+
+```http
+--------------------------d8c247122f594ba0
+Content-Disposition: form-data; name="team_id"
+1
+--------------------------d8c247122f594ba0
+Content-Disposition: form-data; name="install_script"
+sudo installer -pkg /temp/FalconSensor-6.44.pkg -target /
+--------------------------d8c247122f594ba0
+Content-Disposition: form-data; name="pre_install_query"
+SELECT 1 FROM macos_profiles WHERE uuid='c9f4f0d5-8426-4eb8-b61b-27c543c9d3db';
+--------------------------d8c247122f594ba0
+Content-Disposition: form-data; name="post_install_script"
+sudo /Applications/Falcon.app/Contents/Resources/falconctl license 0123456789ABCDEFGHIJKLMNOPQRSTUV-WX
+--------------------------d8c247122f594ba0
+Content-Disposition: form-data; name="software"; filename="FalconSensor-6.44.pkg"
+Content-Type: application/octet-stream
+<BINARY_DATA>
+--------------------------d8c247122f594ba0
+```
+
+##### Default response
+
+`Status: 200`
+
+
+### Download software
+
+_Available in Fleet Premium_
+
+Download software package.
+
+`GET /api/v1/fleet/software/package/:id?alt=media`
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                      |
+| ----            | ------- | ---- | --------------------------------------------     |
+| id              | integer | path | **Required**. The ID of the software title to download software package.|
+| alt             | integer | path | **Required**. If specified and set to "media", downloads the specified software package. |
+
+#### Example
+
+`GET /api/v1/fleet/software/package/123?alt=media`
+
+##### Default response
+
+`Status: 200`
+
+```http
+Status: 200
+Content-Type: application/octet-stream
+Content-Disposition: attachment
+Content-Length: <length>
+Body: <blob>
+```
+
+### Delete software
+
+_Available in Fleet Premium_
+
+Delete software package.
+
+`DELETE /api/v1/fleet/software/package/:id`
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                      |
+| ----            | ------- | ---- | --------------------------------------------     |
+| id              | integer | path | **Required**. The ID of the software title to delete software package. |
+
+#### Example
+
+`DELETE /api/v1/fleet/software/package/24`
+
+##### Default response
+
+`Status: 204`
+
+### Get installation results
+
+_Available in Fleet Premium_
+
+`GET /api/v1/fleet/software/install/results`
+
+Get an installation result of the software that was installed.
+
+| Name            | Type    | In   | Description                                      |
+| ----            | ------- | ---- | --------------------------------------------     |
+| installation_uuid | string | path | **Required**. The installation UUID of the software.|
```

Comment:
```suggestion
| install_uuid | string | path | **Required**. The installation UUID of the software.|
```

Name of the param should be modified to match the corresponding API response field.